src/compressor/QatAccel.cc

```diff
@@ -32,27 +40,23 @@ bool QatAccel::init(const std::string &alg) {
   if (rc != QZ_OK)
     return false;
   params.direction = QZ_DIR_BOTH;
-  if (alg == "snappy")
-    params.comp_algorithm = QZ_SNAPPY;
-  else if (alg == "zlib")
+  if (alg == "zlib")
     params.comp_algorithm = QZ_DEFLATE;
-  else if (alg == "lz4")
-    params.comp_algorithm = QZ_LZ4;
```

Comment:
should explain why remove the snappy & lz4 support in commit message. I know current QAT hardware could not support these algs, and next version will support them this year.

Reply:
In addition, the commit message for this commit should include the list of main issues: 1) compile issues about qatzip found; 2) remove snappy & lz4 ... why?

Reply:
Thank you for your reminder. I will revise the commit message based on your suggestion.